docs/admin/install.rst

```diff
@@ -0,0 +1,265 @@
+=================================================
+
+.. include:: ../includes/top-warning.rst
+
+Prerequisites
+-------------
+In order to install SecureDrop Workstation and configure it to use an existing SecureDrop instance, you will need the following:
+
+- A Qubes-compatible computer with at least 16GB of RAM (32 GB is recommended). SecureDrop Workstation has mainly been tested against Lenovo 6th-gen T470 and X1 models, but there are no formally supported hardware recommendations at this time. See Qubes' `Hardware Compatibility List <https://www.qubes-os.org/hcl/>`_ for more options.
+- Qubes installation medium - this guide assumes the use of a USB 3.0 stick. (Qubes may also be installed via optical media, which may make more sense depending on your `security concerns <https://www.qubes-os.org/doc/install-security/>`_.
+- The SecureDrop instance's Admin Workstation and Secure Viewing Station (SVS) USBs, and the full GPG fingerprint of the submission key.
+- A working computer (Linux is recommended and assumed in this guide) to use for verification and creation of the Qubes installation medium.
+- A password manager or other system to generate and store strong passphrases for Qubes full disk encryption (FDE) and user accounts.
+
+A basic knowledge of the Qubes OS is helpful. 
+
+
+Apply BIOS updates and check settings
+-------------------------------------
+Before beginning the Qubes installation, make sure that your Qubes-compatible computer's BIOS is updated to the latest available version. Instructions on how to do so will vary by manufacturer or model and can usually be found on their respective support sites, for example `https://pcsupport.lenovo.com/ <https://pcsupport.lenovo.com/>`_ in the case of Lenovo.
+
+Once the BIOS is up-to-date, boot into the BIOS setup utility and verify that:
+
+- clock settings are correct
+- virtualization support is enabled
+  - for Intel-based devices, **Intel VT-d** and **Intel VT-x** should be enabled
+  - for AMD-based devices, **AMD-VI** and **AMD-V** should be enabled
+- SecureBoot is disabled.
+
+If the Qubes hardware compatibility list entry for your computer recommends the use of Legacy Mode for boot, change that setting in the BIOS as well.
+
+Download and verify Qubes OS
+----------------------------
+On the working computer, download the latest stable Qubes OS ISO (``4.0.3`` at time of writing) from `https://www.qubes-os.org/downloads/ <https://www.qubes-os.org/downloads/>`_.
+
+Follow the linked instructions to `verify the ISO <https://www.qubes-os.org/security/verifying-signatures/#how-to-verify-qubes-iso-signatures>`_.
+
+Once you've verified the ISO, copy it to your installation medium - for example, if using Linux and a USB stick, using the command:
+
+.. code-block:: sh
+ 
+  sudo dd if=Qubes-R4.0.3-x86_64.iso of=/dev/sdX bs=1048576 && sync
+
+where ``if`` is set to the path to your downloaded ISO file and ``of`` is set to
+the block device corresponding to your USB stick. 
```

Comment:
If we duplicate these instructions, I would include the same caveat here that Qubes does:
> Make sure to write to the entire device (e.g., /dev/sdc) rather than just a single partition (e.g., /dev/sdc1).

Reply:
:heavy_check_mark: